<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DOCFormatter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>main_icon.ico</normaloff>main_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>974</width>
         <height>623</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Formatter">
           <attribute name="title">
            <string>Analysis</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
               </font>
              </property>
              <property name="title">
               <string>Viewer</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QPushButton" name="pushButton_clear">
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>9</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Log Clear</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Data Count</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_count">
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(0, 124, 181);
color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit_print">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(60, 121, 182);</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_1">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>281</width>
                <height>561</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
               </font>
              </property>
              <property name="title">
               <string>Controller</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QGroupBox" name="groupBox_9">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Input Data</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QTextEdit" name="textEdit_browser">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>35</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_browser">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>맑은 고딕</family>
                      <pointsize>11</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Browser</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>150</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Control Button</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QPushButton" name="pushButton_start">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>맑은 고딕</family>
                      <pointsize>11</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="autoFillBackground">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Start</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_cancel">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>맑은 고딕</family>
                      <pointsize>11</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_progress">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Progress</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
